v1/translate.yaml

```diff
@@ -0,0 +1,254 @@
+info:
+  version: '1.0.0'
+  title: Content Translation API
+  description: Content Translation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /machine/{from}/{to}{/provider}:
```

Comment:
Another option could be to extend [the pre-existing `transform` hierarchy](https://en.wikipedia.org/api/rest_v1/#/Transforms), for example via `/transform/language/{from_lang}/to/{to_lang}`.